tests/x11/ghostscript.pm

```diff
@@ -45,6 +45,16 @@ sub run {
     assert_script_run "test -f $gs_script";
     assert_script_run "test -s $gs_script";
 
+    # Remove known failing test file poo#41393/bsc#1109788
+    my $excluded_file = "/usr/share/ghostscript/9.25/examples/transparency_example.ps";
+    my $file_status   = script_run("ls $excluded_file");
+    unless ($file_status) {
```

Comment:
wouldn't it be easier to just skip it in data/ghostscript/ghostscript_ps2pdf.sh ?

I don't think the soft failure reference adds any value (it's generally ignored in QAM workflow)

Reply:
Could be, but i wanted to have soft-fail there. To comment (), i think it will slowly change.

Reply:
I think this will be harder to forget and remove when bug is fixed.

Reply:
But I would just remove the file and add soft_fail, why loop to remove one file ?

Reply:
and if you really need to be root, I would put it [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5834/files#diff-c035f3dd8fb76bf6d0c2700fb2b3cfc7R34)
This are just small things, it can be as it is, but I think does not need to be that much "work" to remove one file :)

Reply:
> I don't think the soft failure reference adds any value (it's generally ignored in QAM workflow)

@coolo at least we have "openqa-review" which puts reminder comments on the bugs with a link to the most recent test failure. I guess this is still some value?